api/apiFunctions.ts

```diff
@@ -1,25 +1,50 @@
+import { Models } from 'appwrite/types/models';
 import { account, databases, ID, appwriteConfig } from './config';
+import { IAccountData, IUserWeeklyPick, IWeeklyPicks } from './IapiFunctions';
 
-export async function loginAccount(user: { email: string; password: string }) {
+/**
+ * Get the current session of the user
+ *
+ * @param email - The email of the user
+ * @param password - The password of the user
+ * @return {Models.Session | Error} - The session object or an error
+ */
+export async function loginAccount({
+  email,
+  password,
+}: IAccountData): Promise<Models.Session> {
   try {
-    return await account.createEmailPasswordSession(user.email, user.password);
+    return await account.createEmailPasswordSession(email, password);
```

Comment:
S - Should `(email, password)` be deconstructed similar to line 126 `const data = { gameId, gameWeekId, userResults };` to stay consistent with formatting and to ensure users are supplying the proper values? 

ex: `const loginInfo = {email, password}`
`return await account.createEmailPasswordSession(loginInfo);

Reply:
Now that you bring it up, line 126 is incorrect. We shouldn't need to create the object because it's not reused. Only when items are reused do you create a variable. You can always refactor the code to be reusable later when it becomes reusable.